src/command_modules/azure-cli-network/azure/cli/command_modules/network/_validators.py

```diff
@@ -1117,6 +1126,44 @@ def process_nw_troubleshooting_show_namespace(cmd, namespace):
     get_network_watcher_from_resource(cmd, namespace)
 
 
+def process_nw_config_diagnostic_namespace(cmd, namespace):
+    from msrestazure.tools import is_valid_resource_id, resource_id
+
+    # validate target resource
+    resource_usage = CLIError('usage error: --resource ID | --resource NAME --resource-type TYPE '
+                              '--resource-group NAME --namespace NAMESPACE [--parent PATH]')
```

Comment:
Can a user include both a namespace and type inside using `--resource-type`? That would be consistent with ARM.

Reply:
I considered that, but none of the other network watcher commands specify namespace, so I wanted to avoid that complexity.